infer.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import os
+import json
+
+from model import ShuffleNet
+
+from torchvision import transforms
+from torch.autograd import Variable
+import torch
+from PIL import Image
+import numpy as np
+
+def get_transformer():
+  normalize = transforms.Normalize(
+      mean=[0.485, 0.456, 0.406],
+      std=[0.229, 0.224, 0.225])
+
+  transformer = transforms.Compose([
+      transforms.Resize(128),
+      transforms.CenterCrop(128),
+      transforms.ToTensor(),
+      normalize
+  ])
+  return transformer
+
+def preprocess(image, transformer):
+  x = transformer(image)
+  return Variable(x.unsqueeze(0))
+
+def infer(args):
+  # make ShuffleNet model
+  print('Creating ShuffleNet model')
+  net = ShuffleNet(num_classes=args.num_classes, in_channels=3)
+  
```

Comment:
The network should be in `.eval()` mode during inference; add `net.eval()` here.

Reply:
Ah that's right. I've made this mistake at least three times--and still don't remember to put it in.